shopfloor/models/stock_move_line.py

```diff
@@ -30,6 +30,7 @@ class StockMoveLine(models.Model):
     picking_id = fields.Many2one(auto_join=True)
 
     quantity_picked = fields.Float(compute="_compute_quantity_picked")
+    is_shopfloor_created = fields.Boolean()
```

Comment:
What is this field for? Should go in separate commit.
I imagine you want mark lines that are create using the app. If yes, make it computed and use a global ctx key set by the base service component.

Reply:
Yes it is to identify which lines were created through shopfloor, as it's not possible to do it anymore only looking at qties as it was the case in previous versions.

I'm ok with your proposal to make it computed, but not sure how do you want to do it.